src/main/java/com/coherentsolutions/training/auto/web/pashkovskaya/pages/AccountCreationPage.java

```diff
@@ -0,0 +1,35 @@
+package com.coherentsolutions.training.auto.web.pashkovskaya.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AccountCreationPage extends BasePage{
+    @FindBy(id = "firstname")
+    public WebElement firstNameTextField;
+    @FindBy(id = "lastname")
+    public WebElement lastNameTextField;
+    @FindBy(id = "email_address")
+    public WebElement emailTextField;
+    @FindBy(id = "password")
+    public WebElement passwordTextField;
+    @FindBy(id = "password-confirmation")
+    public WebElement passwordConfirmationTextField;
+    @FindBy(xpath = "//button [@class='action submit primary']")
+    public WebElement createAccountButton;
+
+    public AccountCreationPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public HomePageAuthorizedUser openHomePageAuthorizedUser(String firstName, String lastName, String email, String password, String passwordConf) {
```

Comment:
let's add more flexibility
1) one action with locator should be in a separate method - single responsibility
2) you never know what else will you have to do with the locator before any action
3) if you want to have signUp method let's just create it as a combination of other methods, example
```
GoogleStartPage
 
Java
public class GoogleStartPage {

  private WebDriver driver;

  @FindBy(id = "searchbar")

  private WebElement searchInput;

  @FindBy(id = "submit")

  private WebElement submitBtn;

  public GoogleStartPage(WebDriver driver) {

    this.driver = driver;

    PageFactory.initElements(this, driver);

  }

  public GoogleSearchResultsPage doSearchQuery(String query) {

    this.setQuery(query);

    return this.submitSearch();

  }

  // implementation
  public boolean isOpened() {

    try {

      return searchInput.isDisplayed();

    } catch (Exception e) {

      return false;

    }

  }

}
```